CMakeLists.txt

```diff
@@ -184,6 +184,9 @@ INSTALL(TARGETS smack
 INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/bin/boogie
   ${CMAKE_CURRENT_SOURCE_DIR}/bin/corral
   ${CMAKE_CURRENT_SOURCE_DIR}/bin/smack.py
+  ${CMAKE_CURRENT_SOURCE_DIR}/bin/tokenizer
```

Comment:
This is a binary provided by SVCOMP for converting C files to have one token per line.  I don't know of a better place to put this than bin, since it is not really part of smack...

I suppose we could put it in the `share/` installation directory, and reference it using relative paths...